### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by BlackRockJL** [niklaswa/TrainStatusInfo#2]
Does this still work with a current macOS and how can I run the application? swift main.swift seems not to work. :)

**Comment by niklaswa** [comment]
Yes, it should still work. You have to clone it in Xcode and run it from there.

I also just build a signed & notarized version, if you really want to run it (even without Xcode): 
[TrainStatusInfo.zip](https://github.com/niklaswa/TrainStatusInfo/files/14042385/TrainStatusInfo.zip)

Btw, there is a similar, more refined app in the app store by another developer which has even more features: https://apps.apple.com/de/app/ice-buddy-for-your-menu-bar/id1595947689
